src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -167,6 +167,9 @@ function AnnotationEditor({
 
   const textStyle = applyTheme(['annotationFontFamily'], settings);
 
+  const mentionsEnabled = store.isFeatureEnabled('at_mentions');
+  const usersWhoAnnotated = store.usersWhoAnnotated();
```

Comment:
The list of suggested users also includes the current user. That feels a bit strange. I note that GitHub does include `@robertknight` in the list of suggestions as I type this comment. It is possible to see yourself in X and Bluesky mentions too. That surprised me.

<img width="427" alt="self at-mention" src="https://github.com/user-attachments/assets/419d1db1-1bb5-43c4-bd3e-d9fdbbde66bb" />

Reply:
It shouldn't be super hard to filter out "self" later, if we decide that's the desired behavior.